docs/reference/commandline/daemon.md

```diff
@@ -454,6 +454,14 @@ This example sets the `cgroupdriver` to `systemd`:
 
 Setting this option applies to all containers the daemon launches.
 
+Also Windows Container makes use of `--exec-opt` for special purpose. Docker user
+can specify default container isolation technology with this, for example:
+
+    $ sudo docker daemon --exec-opt isolation=hyperv
```

Comment:
Yes, you are right?
@jhowardmsft How to fix this? Maybe I can simply remove `sudo` because Windows Docker has same UX with Linux Docker?